mon-pix/app/components/challenge-item-generic.js

```diff
@@ -62,29 +51,15 @@ export default class ChallengeItemGeneric extends Component {
     }
   }
 
-  _start() {
-    this._elapsedTime = 0;
-    this._tick();
-  }
-
-  _tick() {
-    if (ENV.APP.isChallengeTimerEnable) {
-      const timer = later(this, function() {
-        const elapsedTime = this._elapsedTime;
-        this._elapsedTime = elapsedTime + 1;
-        if ((this._elapsedTime - this.args.challenge.timer) >= 0) {
-          this.hasChallengeTimedOut = true;
-        } else {
-          this._tick();
-        }
-      }, 1000);
-
-      this._timer = timer;
-    }
```

Comment:
**question (blocking):** En supprimant le tick et avec la méthode suivie, nous perdons l'information du temps passé sur la question. 
- Est-ce que cette information est exploitée par le contenu ? 
- Si oui, est-ce qu'on essayerait de récupérer le temps passé depuis le timer et déplacer la responsabilité au challenge de savoir si le temps restant fait timeout ou non au risque de dupliquer la gestion du timeout (j'ai dû mal à avoir une vision claire sur ce chemin sans code concret) ? 
- Si non, est-ce qu'il ne faut pas inclure dans cette PR la reconception de `_getTimeout` pour retourner un boolean au lieu d'un nombre à `this.args.answerValidated` ?

Reply:
Il y a `_elapsedTime` qui représente le temps passé sur la question.
Par contre en supprimant le timer, cette valeur n'est plus mise à jour => la méthode `_getTimeout` ne doit plus marcher.
Je vais ajouter une méthode à TimeoutGauge pour qu'il puisse retourner combien de temps a été passé sur la question.